packages/lit-helpers/test/live.test.js

```diff
@@ -0,0 +1,132 @@
+import { expect, html, fixture } from '@open-wc/testing';
+import { spy } from 'sinon';
+import { render } from 'lit-html';
+import { live } from '../src/live.js';
+
+class LitHelpers extends HTMLElement {
+  set myProp(value) {
+    this._myProp = value;
+  }
+
+  get myProp() {
+    return this._myProp;
+  }
+}
+
+customElements.define('lit-helpers', LitHelpers);
+
+describe('live', () => {
+  describe('property bindings', () => {
+    let wrapper;
+    beforeEach(async () => {
+      wrapper = await fixture(document.createElement('div'));
+    });
+
+    function renderLive(value) {
```

Comment:
probably should go in a before?

Reply:
I don't think so, it's just a function definition.